routes/profile.js

```diff
@@ -0,0 +1,40 @@
+var express = require('express');
+var router = express.Router();
+var mongoProfile = require('../models/Profile').mongoProfile;
+var joiProfile = require('../models/Profile').joiProfile;
+
+    router.get('/:id', async (req, res) => {
+
+        try {
+            let id = req.params.id;
+            let Profile = await mongoProfile.findById(id);
+            res.send(Profile);
+        } 
+        catch(err) {
+            res.status(400).send({status: "Failed", message: "Cannot get data"})
+        }
+    
+    })
```

Comment:
keep indent consistent
```suggestion
router.get('/:id', async (req, res) => {

    try {
        let id = req.params.id;
        let Profile = await mongoProfile.findById(id);
        res.send(Profile);
    } 
    catch(err) {
        res.status(400).send({status: "Failed", message: "Cannot get data"})
    }
    
})
```